pkg/virt-operator/resource/apply/rbac.go

```diff
@@ -215,9 +216,51 @@ func getRoleMetaObject(role interface{}, roleType RoleType) (meta *metav1.Object
 	return
 }
 
-func areRoleRulesEqual(role1 interface{}, role2 interface{}, roleType RoleType) (equal bool) {
+func enforceAPIGroup(existing interface{}, required interface{}, roleType RoleType) {
```

Comment:
I'd also add a line in the commit message on why this commit exists rather than just stating what it does (which you can anyway find out by reading the code) because that's not very clear to me as I'm not overly familiar with this area of the code, also it might be useful to others when exploring the git history.